wallet/src/de/schildbach/wallet/ui/preference/PreferenceActivity.java

```diff
@@ -68,7 +68,7 @@ private void initToolbar()
     @Override
     public void onBuildHeaders(final List<Header> target) {
         loadHeadersFromResource(R.xml.preference_headers, target);
-        hidePersonalSettingsIfLocked(target);
+//        hidePersonalSettingsIfLocked(target);
```

Comment:
This is acceptable for now until the WalletLock functions are integrated.

Reply:
I left this line commented out in order to simplify testing process.